frontend/src/components/Main.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+
+import { API_URL } from '../reusable/urls';
+
+const Main = () => {
+  const accessToken = useSelector((store) => store.user.accessToken);
+  const history = useHistory();
+
+  useEffect(() => {
+    if (!accessToken) {
+      history.push('/login');
+    }
+  }, [accessToken, history]);
+
+  useEffect(() => {
+    if (accessToken) {
+      const options = {
+        method: 'GET',
+        headers: {
+          Authorization: accessToken,
+        },
+      };
+
+      fetch(API_URL('mypage'), options)
+        .then((res) => res.json())
+        .then((data) => console.log(data));
```

Comment:
❕❕ There should be no developer console logs in production. Also not sure what the purpose of this fetch function is... Cause all it does is post the data in the console. Maybe you forgot to add something here?

Reply:
🧹 console.log 🧹

Reply:
Yes I didn't have time to give it a reasonable functionality but wanted to leave it for further development.